main/main.c

```diff
@@ -673,12 +673,11 @@ extern int main (int argc CTAGS_ATTR_UNUSED, char **argv)
 	runMainLoop (args);
 
 
-	BEGIN_VERBOSE(vfp);
+	if (Option.verbose || Option.mtablePrintTotals)
 	{
 		for (unsigned int i = 0; i < countParsers(); i++)
-			printLanguageMultitableStatistics (i, vfp);
+			printLanguageMultitableStatistics (i, stderr);
```

Comment:
Done.

BTW, you know instead of doing `BEGIN_VERBOSE();` and `END_VERBOSE();` and have using code do this:

```
	BEGIN_VERBOSE(vfp);
	{
		printInputLine(vfp, current, *offset);
	}
	END_VERBOSE();
```
which is... verbose 😉 , we could instead make a macro like this:

```
#define IF_VERBOSE(VFP) for (FILE* VFP = stderr; Option.verbose && VFP; VFP = NULL)
```
so that using code just looks like this:
```
	IF_VERBOSE(vfp)
	{
		printInputLine(vfp, current, *offset);
	}
```

Reply:
That's a  good idea.